roles/kubernetes/node/templates/kubelet.j2

```diff
@@ -72,3 +72,8 @@ KUBELET_CLOUDPROVIDER="--cloud-provider={{ cloud_provider }}"
 {% else %}
 KUBELET_CLOUDPROVIDER=""
 {% endif %}
+
+{% if ansible_os_family in ['CoreOS', 'Container Linux by CoreOS'] %}
+# set PATH to include /opt/bin (e.g. for socat)
+PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/opt/bin
```

Comment:
Let's not make it a conditional. Let's do this instead without any comments or if statement.
```
PATH={{ bin_dir }}:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
```